frame/types.go

```diff
@@ -109,6 +109,82 @@ const (
 // https://github.com/apache/cassandra/blob/trunk/doc/native_protocol_v4.spec#L502
 type ErrorCode = Int
 
+// See CQL Binary Protocol v4, section 9 for more details.
+// https://github.com/apache/cassandra/blob/trunk/doc/native_protocol_v4.spec#L1046-L1200
+const (
```

Comment:
Is there a need for errors to be located in frame/types.go? We already have them in errors.go